tests/common/plugins/conditional_mark/tests_mark_conditions.yaml

```diff
@@ -27,11 +27,17 @@ copp/test_copp.py::TestCOPP::test_trap_config_save_after_reboot:
 #######################################
 #####            decap            #####
 #######################################
+decap/test_decap.py::test_decap[ttl=pipe, dscp=pipe]:
+  skip:
+    reason: "Not supported on broadcom after 201911 release and mellanox all releases"
+    conditions:
+      - "(asic_type in ['broadcom'] and release in ['202012', 'master']) or (asic_type in ['mellanox'])"
```

Comment:
There may new release after 202012, could we use not in ['201811', '201911']?

Reply:
Sure, that make sense.